### PR TITLE
ECRecover Proof of Concept

### LINKED ISSUE #167: Investigate limitation of eth public address
### Problem

Currently eth ecosystem uses the public address (which is the last 20 bytes of the Keccak-256 hash of the public key with a 0x prefix) to identify accounts.
However, Hedera requires the public key not the address to serve as an alias for account creation and reference.
This makes it challenging to use the public address as is desired

### Solution

Explore the limitations and possibilities of using the public address as
- an alias for account creation
- a reference during transactions

Deliverables would be
- Document the limitations
- Open tickets for the possibilities

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by shemnon**
Proof of concept to demonstrate ECRECOVER precompile

Signed-off-by: Danno Ferrin <danno.ferrin@hedera.com>

**Description**:

**Related issue(s)**:

Fixes #167 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by Nana-EC** [comment]
Wondering if we should move this over to the `smart-contracts` repo and reference in documentation here that wallet users may utilize this as one of the many flows to extract the public key.

File: dapp-example/src/ExtractPublicKey/EtractPublicKey.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
```
Comment (natanasow) [inline dapp-example/src/ExtractPublicKey/EtractPublicKey.html:1]: nit: typo in the file name `EtractPublicKey.html`

**Review by natanasow — APPROVED** [review]
LG

**Comment by Nana-EC** [comment]
Will take over moving this to appropriate location in this repo or in the Smart Contracts repo.

**Pull request closed without merging**